src/hotspot/share/runtime/synchronizer.hpp

```diff
@@ -143,10 +143,10 @@ class ObjectSynchronizer : AllStatic {
   static size_t deflate_idle_monitors();
 
   // Deflate idle monitors:
-  static void chk_for_block_req(JavaThread* self, const char* op_name,
+  static void chk_for_block_req(JavaThread* current, const char* op_name,
```

Comment:
I like "current" better than "self" as a convention.

Reply:
Out of all the changes from "self" -> "current"... how did you decide to land
a comment on this one... :-)